### DISCUSSION TITLE
Linux Template user local can´t access

### DISCUSSION THREAD

**Discussion by pere-pou** [vmware/packer-examples-for-vsphere#477] [Help] [answered]
hi all! when i launch the packer to create an ubuntu template everything great. is amazing! 
but The problem is that we cant access directly this machine with the edited username and password in build.pkrvars.hcl

We can access by ssh in areas where I have DCHP, but if not, the edited credentials don't work for me... do you know why?

Is there a configuration that does not enable the login directly? in the file build.pkrvars.hcl

**Comment by tenthirtyam** [comment]
> hi all! when i launch the packer to create an ubuntu template everything great. is amazing! but The problem is that we cant access directly this machine with the edited username and password in build.pkrvars.hcl
> 
> We can access by ssh in areas where I have DCHP, but if not, the edited credentials don't work for me... do you know why?
> 
> Is there a configuration that does not enable the login directly? in the file build.pkrvars.hcl

Are you saying that you're unable to login to the virtual machines guest OS using the remote console or web console, but that you can using SSH?

**Reply by pere-pou** [reply]
i was deployed ubuntu 22-04-lts. I will try with yours explamples files.
There willnot be a password restrictions or a rule?(number characters, special symbols...)
with windows deploys it works perfectly

**Reply by tenthirtyam** [reply]
The password complexity would be based on the Linux pam.d configuration.

As seen in the example, it uses a more complex 17-character password with uppercase, lowercase, numbers, and symbols and you issue _may_ be guest OS specific.

**Comment by pere-pou** [comment]
yes, with ssh i can access with my key that i was configured in the build.pkrvars.hcl but when i try access to remote console is not possible.

**Comment by xveral** [comment]
Same issue here! vmware remote console does not login; but using ssh with key works with no issues. Build file seems to be ok because we use it to deploy windows server vms and it works. (user/pwd applied correctly). Seems like pwd is 'deployed' incorrectly; thanks in advance!

**Comment by tenthirtyam** [comment, marked as answer]
Interesting, I've never had an issue with the example `rainpole` user.

**Reply by tenthirtyam** [reply]
@pere-pou @xveral - I've not been able to reproduce this concern. Assume the same result with Remote Console _and_ Web Console?

**Reply by pere-pou** [reply]
yes, i can access via ssh(configured my keys) but when i try access web console nop, after access ssh and restart password user, i can access. I don't see what could be happening, I even used the user that you gave as an example in the code

**Reply by tenthirtyam** [reply]
For a test, can you try the example password to see if the results are the same?

**Reply by xveral** [reply]
Sorted out! The problem was a capital letter in the username, seems ubuntu didn't like it.  :-)

**Reply by tenthirtyam** [reply]
Very interesting! :)
